.travis.sh

```diff
@@ -37,14 +37,36 @@ install_from_github() {
 }
 
 sudo apt-get update -qq
+
 # libpcsclite-dev is required for OpenSC
 sudo apt-get install -y libpcsclite-dev
 
 export CC=`which $CC`
 mkdir prerequisites
 cd prerequisites
+
+# Install OpenSSL
+if [ -n "${OPENSSL}" ]; then
+    # Remove pre-installed OpenSSL
+    sudo apt-get remove openssl libssl-dev
+
+    OPENSSL_DIR=/usr
+    git clone https://github.com/openssl/openssl.git -b ${OPENSSL}
```

Comment:
please add ` --depth=1`.

This optimization should also be used in `install_from_github`. `install_from_github` could even be split up into `install_from_github`, which calls `fetch_from_github`, where the latter can also be used for OpenSSL...

Reply:
Applied the suggested changes in 75fa772